compiler-core/src/docs.rs

```diff
@@ -198,21 +201,35 @@ pub fn generate_html(
             url: format!("{}.html", module.name),
         });
 
+        let rendered_documentation = render_markdown(&module.ast.documentation.iter().join("\n"));
+        let rendered_text_documentation =
+            manipulate::trim(rendered_documentation._strip_tags().as_str(), "\n ");
+        let page_meta_description = &manipulate::replace_all(
```

Comment:
Is there a better/more elegant way than to use `voca_rs` crate?

Reply:
What does this do?

Reply:
It creates the meta description from the module documentation doc block?